cdap-docs/admin-manual/source/security/secure-storage.rst

```diff
@@ -42,16 +42,55 @@ CDAP provides two different implementations of secure storage, depending on the
 
 File-backed Secure Storage
 --------------------------
-File-backed secure storage is available for use with in-memory (unit-test) and standalone (CDAP SDK) modes. It uses the
+File-backed secure storage is available for use with in-memory CDAP (unit-test) and
+standalone CDAP (as included in the CDAP SDK) modes. It uses the
 `Sun JCEKS <http://docs.oracle.com/javase/7/docs/technotes/guides/security/crypto/CryptoSpec.html#KeyManagement>`__
 implementation for storing secure keys. This implementation is not available in
 :term:`Distributed CDAP <distributed cdap>` as it stores the secure data in the local file system, and thus is
 not available on all nodes of a distributed cluster.
 
 To use this mode, set these properties:
 
-- In ``cdap-site.xml``, set ``security.store.provider`` to ``file``
-- In ``cdap-security.xml``, set ``security.store.file.password`` to a password (to protect the secure storage file)
+- In ``cdap-site.xml``, set ``security.store.provider`` to ``file``::
+
+    <property>
+      <name>security.store.provider</name>
+      <value>file</value>
+      <description>
+        Backend provider for the secure store
+      </description>
+    </property>
+
+- In ``cdap-security.xml``, set ``security.store.file.password`` to a password (to protect the secure storage file)::
+
+    <property>
+      <name>security.store.file.password</name>
+      <value>your password</value>
+      <description>
+        Password to access the key store
+      </description>
+    </property>
+    
+- The path and the filename of the backing file can be configured in ``cdap-security.xml``
```

Comment:
Following two properties go in cdap-site.xml, only password goes in cdap-security.xml